sdk/js/src/solana/wormhole/coder/idl.ts

```diff
@@ -0,0 +1,112 @@
+// Borrowed from coral-xyz/anchor
```

Comment:
Just making a comment here as a note that we would probably like this for Token Bridge and NFT Bridge program decoding.